handlers.go

```diff
@@ -19,11 +19,13 @@ import (
 	"bytes"
 	"encoding/base64"
 	"fmt"
+	"github.com/coreos/go-oidc/oauth2"
```

Comment:
So a go idiom is local packages at top, local package in the middle and external at the bottom